pkg/scoring/http_error.go

```
@@ -0,0 +1,53 @@
+	RateLimited   int
+	NetworkErrors int
+	ServerErrors  int
+}
+
+// Any reports whether any modifier errors occurred during the scan.
+func (s HTTPModifierStats) Any() bool {
+	return s.Timeouts > 0 || s.RateLimited > 0 || s.NetworkErrors > 0 || s.ServerErrors > 0
+}
+
+// any is a package-internal alias for Any.
+func (s *HTTPModifierStats) any() bool {
```

golangci-lint (2.11.4)

[error] func (*HTTPModifierStats).any is unused

(unused)